packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -107,7 +107,7 @@ const Feature2TemplateName: any = {
   [`${CapabilityOptions.commandBot().id}:undefined`]: TemplateNames.CommandAndResponse,
   [`${CapabilityOptions.workflowBot().id}:undefined`]: TemplateNames.Workflow,
   [`${CapabilityOptions.basicBot().id}:undefined`]: TemplateNames.DefaultBot,
-  [`${CapabilityOptions.me().id}:undefined`]: TemplateNames.MessageExtension,
+  [`${CapabilityOptions.me().id}:undefined`]: TemplateNames.MessageExtensionAction,
```

Comment:
Based on below, should here be `CapabilityOptions.collectFormMe().id` ?https://github.com/OfficeDev/TeamsFx/blob/dea6b9e8aae05e82dca0dedecfadd0fd22ea591f/packages/fx-core/src/question/create.ts#L391

I think the reason it still works is that `CapabilityOptions.collectFormMe()` and `CapabilityOptions.me()` share the same id.

Reply:
Yes, I agree. Thanks.